src/plugins/intel_gpu/tests/functional/single_layer_tests/dynamic/convolution.cpp

```diff
@@ -88,13 +88,6 @@ class ConvolutionLayerGPUTestDynamic : public testing::WithParamInterface<convLa
         size_t convOutChannels;
         std::tie(kernel, stride, padBegin, padEnd, dilation, convOutChannels, padType) = convParams;
```

Comment:
could you please explain in the commit message "why it is oK to remove the code"?

It would be something like "It was added by mistake. It is possible to detect failure without changing threshold."

Reply:
yes, it's mistake. That setting threshold manually occurs some test-failures on dGPU.